src/platform/plugins/shared/chart_expressions/expression_xy/public/helpers/annotations.tsx

```diff
@@ -221,3 +225,22 @@ export function Marker({
   }
   return null;
 }
+
+const xyDecorationRotatedWrapperStyles = css({
+  display: 'inline-block',
+  overflow: 'hidden',
+  lineHeight: 1.5,
+
+  '& .xyDecorationRotatedWrapper__label': {
```

Comment:
Is the `&` necessary here? In this usage, wouldn't it be the same as if it where just targeting the child class?

```suggestion
  '.xyDecorationRotatedWrapper__label': {
```

Reply:
I think is not necessary but using `& .xyDecorationRotatedWrapper__label` ensures that the styles are applied only to elements with the class `.xyDecorationRotatedWrapper__label` that are descendants of the current element